sonar-python-plugin/src/main/java/org/sonar/plugins/python/caching/Caching.java

```diff
@@ -62,6 +63,11 @@ public void writeImportsMapEntry(String moduleFqn, Set<String> imports) {
     cacheContext.getWriteCache().write(cacheKey, importData);
   }
 
+  public void writeFileContentHash(String fileKey, byte[] hash) {
+    String cacheKey = CONTENT_HASHES_KEY + fileKey;
```

Comment:
Would it make sense to normalize the cache key here?
```
String cacheKey = CONTENT_HASHES_KEY + fileKey.replace('\\', '/');
```

Reply:
Thanks! Indeed we probably want to do that.
Note that other keys were not normalized either, I handled all of them in a dedicated commit.